Source/Revit.IFC.Import/Importer.cs

```diff
@@ -439,8 +439,10 @@ private bool DocumentUpToDate(Document doc, string ifcFileName)
       /// <param name="document">The host document for the import.</param>
       /// <param name="origFullFileName">The full file name of the document.</param>
       /// <param name="options">The list of configurable options for this import.</param>
-      public void ReferenceIFC(Document document, string origFullFileName, IDictionary<String, String> options)
+      public void ReferenceIFC(Document document, string origFullFileName, IDictionary<String, String> options, out ElementId revitLinkTypeId)
       {

```

Comment:
Would it be better to just return the ElementId instead?

Reply:
that was my first idea, but I've thought that changing the return method from void to ElementId could it be potentially more distruptive for existing codes which uses this DLL rather than leaving to void and using an out reference

Reply:
Hmm, I guess it depends if it is overridden or not.  I would think changing void to ElementId as a return wouldn't be so bad, would it?  Vs. changing the signature?

Reply:
well, returning ElementId it would be cleaner and more straightforward too, if you think that's ok for compatibility I'd also prefer that way,

do you want me to create another pull request with the implementation?